Examples-Web/HoverButton/BasicHoverButton.html

```diff
@@ -1,15 +1,42 @@
 ﻿<!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <title>Basic Hover Button Example</title>
-        <script name="TouchFree_Tooling" type="text/javascript" src="./TouchFree/TouchFree_Tooling.js"></script>
-        <script type="text/javascript" src="./BasicHoverButton.js"></script>
-        <link rel="stylesheet" href="./BasicHoverButton.css" />
-    </head>
-
-    <body class="bodyBG">
-        <button class="button" style="background-color: #fdac53" />
-        <button class="button" style="background-color: #d2386c" />
-    </body>
-</html>
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Basic Hover Button Example</title>
+    <script name="TouchFree_Tooling" type="text/javascript" src="./TouchFree/TouchFree_Tooling.js"></script>
+    <link rel="stylesheet" href="./BasicHoverButton.css" />
+</head>
+
+<body>
+    <button class="button" style="background-color: #fdac53" onpointerenter="onHover(this)"
+        onpointerleave="onUnhover(this)" onpointerdown="onDown(this)" onpointerup="onUp(this)">
```

Comment:
👀 If these were arrow functions then we don't need to bind (this)

Reply:
Turns out you still do! We think this is something we're confusing with React behaviour.